index.js

```diff
@@ -26,6 +26,14 @@ module.exports = {
         */
         'comma-dangle': ['error', 'only-multiline'],
 
+        /*
+            Always provide a `default` case in `switch` statements.
+            If the default case is logically unreachable, throw an Error.
+            If the default case should do nothing, add a comment explaining why
+            to indicate that it is an intentional decision.
+        */
+        // Notes for the Airbnb 'default-case' rule configuration
```

Comment:
What's the point of this comment and the one for `no-floating-promises` below?

Reply:
I merged the comment showing the rule name and the comment with the rule note together for these two cases.